components/Buttons/examples/ButtonsTypicalUse.m

```diff
@@ -43,6 +43,8 @@ - (MDCButton *)buildCustomStrokedButton {
 - (void)viewDidLoad {
   [super viewDidLoad];
 
+  id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] init];
```

Comment:
I typically expect protocols in the context of interfaces, methods and delegate.
It seems strange to have it as the sole reference to a concrete object.
Consider changing to:
MDCSemanticColorScheme *colorScheme = [[MDCSemanticColorScheme alloc] init];

Reply:
done